src/Kliva/Views/ActivityDetailPage.xaml

```diff
@@ -28,27 +28,10 @@
         </CommandBar>
     </Page.BottomAppBar>
 
-    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
-        <VisualStateManager.VisualStateGroups>
-            <VisualStateGroup x:Name="VisualStateGroup">
-                <VisualState x:Name="Mobile">
-                    <VisualState.StateTriggers>
-                        <AdaptiveTrigger MinWindowWidth="320" />
-                    </VisualState.StateTriggers>
-                    <VisualState.Setters>
-                        <Setter Target="BottomAppBar.Visibility" Value="Visible" />
```

Comment:
Actually I don't think there's a problem here? The AppBar is now Visible by default.